gensim/corpora/wikicorpus.py

```diff
@@ -612,6 +613,8 @@ def __init__(self, fname, processes=None, lemmatize=utils.has_pattern(), diction
             If set, each XML article element will be passed to this callable before being processed. Only articles
             where the callable returns an XML element are processed, returning None allows filtering out
             some articles based on customised rules.
+        metadata: bool, optional
+                if True - write article titles to corpus
```

Comment:
What "corpus"? Please make the docstring more explicit, less cryptic (and properly capitalized and punctuated, like the others).

Reply:
You can edit PRs in place – no need to open a new PR for each change.

Reply:
Thanks for being patient with my first PR (ever!).
Updated documentation.

_"What "corpus"? Please make the docstring more explicit, less cryptic (and properly capitalized and punctuated, like the others)."_
- Updated comment (copied an existing comment describing 'metadata' parameter.